resource_customizations/numaplane.numaproj.io/MonoVertexRollout/actions/unpause-gradual/action.lua

```diff
@@ -1,3 +1,11 @@
 obj.spec.monoVertex.spec.lifecycle.desiredPhase = "Running"
 obj.spec.monoVertex.spec.replicas = null
```

Comment:
There's no longer a reason to set `obj.spec.monoVertex.spec.replicas = null` so let's remove that